.github/scripts/snapshot_images.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Initialize array to store digest SHAs
+declare -a digests
+
+# Fetch all docker image names
```

Comment:
A: Since we already rely on the release version being passed from the `release_images.sh` script, we could also pass the image names so we don't have to query for them twice.